src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Physical Cluster Replication Technical Overview
+summary: Understand the technical details of CockroachDB physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication uses two clusters with _cluster virtualization_ enabled to stream data from an active primary cluster to a passive standby cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Start the replication stream
+
+Starting a physical replication stream consists of two jobs: one on the primary and one on the standby. The job on the standby cluster communicates with the primary cluster via an ordinary SQL connection and is responsible for initiating the replication stream. The stream initialization proceeds as follows:
```

Comment:
What if you named the jobs here? "The physical stream producer job runs on the primary cluster, and the ??? job runs on the standby cluster." And then describe each job? I don't think you mention what the standby cluster's job is.

Reply:
Added, I think I captured what you were looking for here.